v1/api/swagger/components/schemas/UpdateVolume.yaml

```diff
@@ -0,0 +1,46 @@
+allOf:
```

Comment:
Capacity is the only field that can be updated. UpdateVolume should only have fields updatable and fields required like Etag. right?

Reply:
Yes I agree. This is currently the model used for update though.